akka-remote/src/test/scala/akka/remote/artery/compress/HandshakeShouldDropCompressionTableSpec.scala

```diff
@@ -89,17 +88,21 @@ class HandshakeShouldDropCompressionTableSpec extends AkkaSpec(HandshakeShouldDr
       Thread.sleep(5000)
```

Comment:
I'll try to remove in my next PR, wanted a delay to see "the system is actually down", before binding again to same address